securedrop/i18n.py

```diff
@@ -49,7 +49,9 @@ def setup_app(app, translation_dirs=None):
     app.config['BABEL_TRANSLATION_DIRECTORIES'] = translation_dirs
 
     babel = Babel(app)
-    assert len(list(babel.translation_directories)) == 1
+    if len(list(babel.translation_directories)) != 1:
+        raise AssertionError('Multiple babel translation directories found.')
```

Comment:
It could also be an empty list :-) What about:
<pre>
raise AssertionError('Expected exactly one translation directory but got {}.'.format(babel.translation_directories))
</pre>

Reply:
Fixed.